docs/GettingStartedDocs/Contributors/building_oe_sdk.md

```diff
@@ -33,22 +33,25 @@ Please refer to the following [documentation](/docs/GettingStartedDocs/SGXSuppor
 
    On Linux, if your target system does not have any SGX hardware support, you may want to choose simulation.
 
-   On Windows, Open Enclave SDK does not support simulation mode.
+   On Windows, Open Enclave SDK supports simulation mode but only on systems with SGX1 or SGX1+FLC support.
```

Comment:
To confirm: Did we verify there's a difference between Linux and Windows for simulation mode?  I thought at one point someone said it was the same on both.   If it's confirmed to be different, then the text is fine.

Reply:
@dthaler, currently investigating this on Windows. Will update directions if I can get it to build on a non SGX machine

Reply:
Have not been able to get the SDK to build on non SGX system in current state. Need PR https://github.com/openenclave/openenclave/pull/2882 to go in. For v0.9 on Windows, claiming only to support simulation mode on SGX systems.